### PR TITLE
Implement buttons to manually print receipts

### PR DESCRIPTION
**Pull request by chaotixkilla**

### CODE REVIEW

**Comment by chaotixkilla** [comment]
[Receipt printer non-collection](https://trello.com/c/afXrEWak/706-receipt-printer-non-collection)

File: lib/brain.js [hunk 1]
```diff
@@ -3052,10 +3054,15 @@ Brain.prototype._cashInComplete = function _cashInComplete () {
...
-    this._printReceipt()
-  }
+Brain.prototype._startPrintReceipt = function _startPrintReceipt () {
+  this.browser().send({ receiptStatus: 'printing' })
+  this._printReceipt()
+
+  setTimeout(() => {
+    this.browser().send({ receiptStatus: 'success' })
+  }, 1000)
 }
```
Comment (josepfo) [inline lib/brain.js:3066]: Does this mean that we present a success message even if there's an error with the printer? If this is the case, maybe we should inform that something went wrong in the UI. Let me know what you think.

**Review by josepfo — APPROVED** [review]
Looks good!

File: i18n/ui/lbm-ui_en-US.po [hunk 2]
```diff
@@ -102,6 +102,14 @@ msgstr "You Purchased"
...
 msgstr "Your coins are on their way to:"
 
+#: On screen: completed
+msgid "Print receipt"
+msgstr "Print receipt"
+
+#: On screen: completed
+msgid ""
+msgstr ""
+
```
Comment (RafaelTaranto) [inline i18n/ui/lbm-ui_en-US.po:112]: Hmm, looks like something useless has a i18n tag

File: i18n/ui/lbm-ui_en-US.po [hunk 3]
```diff
@@ -475,8 +483,8 @@ msgid "Please press the back of your driver's license against the scan window."
 msgstr "Please press the back of your driver's license against the scan window."
 
 #: On screen: scan-photo-manual
-msgid "Please press the front of your state ID card against the scan window, hold it there, and then press \"Ready to scan\"."
-msgstr "Please press the front of your state ID card against the scan window, hold it there, and then press \"Ready to scan\"."
+msgid "Please press the front of your state ID card against the scan window, hold it there, and then press "Ready to scan"."
+msgstr "Please press the front of your state ID card against the scan window, hold it there, and then press "Ready to scan"."
```
Comment (RafaelTaranto) [inline i18n/ui/lbm-ui_en-US.po:487]: Escape bs

**Review by RafaelTaranto — CHANGES_REQUESTED** [review]

File: i18n/ui/lbm-ui_en-US.po (hunk 2 above)
Comment (chaotixkilla) [reply i18n/ui/lbm-ui_en-US.po:112]: Oops, you're right

**Review by RafaelTaranto — APPROVED** [review]

**Pull request merged**